app/services/content_block/find_and_replace_embed_codes_service.rb

```diff
@@ -7,11 +7,11 @@ def self.call(html)
     def call
       embed_content_references.uniq.each do |reference|
         content_block = content_blocks.find do |c|
-          c.document.content_id == reference.identifier || c.document.content_id_alias == reference.identifier
+          c.fetch("content_id_aliases").first.fetch("name") == reference.identifier
```

Comment:
I know we will _usually_ only have one `content_id_alias`, but I wonder if it's more futureproof to do an `includes` here? Something like:

```ruby
c.fetch("content_id_aliases").map { |alias| alias["name"] }.includes?(reference.identifier)
```

Reply:
We don't yet, but the way the data is structured means there might be a case where that does happen. Just thinking in terms of future proofing.